src/api/rest/providers/github/endpoints/orgs.js

```diff
@@ -0,0 +1,65 @@
+import has from 'lodash.has';
+import { CALL_API } from 'api/rest/middleware';
+import {
+  ORGS_GET_BY_ID,
+  REPOS_BY_ORG,
+  MEMBERS_BY_ORG,
+} from 'api/rest/actions/orgs';
+import { handlePaginatedApi } from 'api/rest/providers/github';
+
+import Schemas from '../schemas';
+
+const _getById = orgId => ({
+  [CALL_API]: {
+    types: ORGS_GET_BY_ID,
+    endpoint: `orgs/${orgId}`,
+    schema: Schemas.ORG,
+  },
+});
+
+export const getById = (
+  orgId,
+  { requiredFields = [], forceRefresh = false } = {}
+) => (dispatch, getState) => {
+  const org = getState().entities.orgs[orgId];
+
+  if (!forceRefresh && org && requiredFields.every(key => has(org, key))) {
+    return null;
+  }
+
+  return dispatch(_getById(orgId));
```

Comment:
I avoided using an anonymous function for now, so that the code is understandable.

The proposed renaming is: 
 - exported function: orgsGetById()
 - private function getById() (no need to prefix with orgs, we're inside orgs.js)

Reply:
Agree, but why `orgsGetById` instead of `getOrgsById`?

Reply:
Just pushed a refactoring that namespaces the client like I originally intended.

Now from a screen, you simply import `client` from `providers/github`, then you pass the functions you need as props in `connect()`, renaming them as you want for the screen code.

See here: https://github.com/machour/git-point/blob/82495a62accfc4b7f383bcb3e5862592a3efd7ff/src/screens/organization/repository-list.js#L94

Does that sound better to you?

Reply:
Now, of course! At once it became much clearer, you too?

Reply:
And still, I do not really like that the spelling of entities is differently used: `orgs` or `organizations`, `repos` or `repositories`. I want to just come to the same spelling, and I would choose the full name, and it used it everywhere.

Reply:
Yes! I was struggling for hours trying to use a Proxy() on the parameters passed to `connect()`, before thinking of doing this trivial thing 😭 

Just like you mentioned for schema, I'd like to have a base client, used only to document endpoints & methods (and to typecheck future implementations later)

Reply:
@lex111 on my side, I'm more tempted to use the short forms everywhere in the app :D 

We'll have a vote on that if needed. #teamShort